README.md

```diff
@@ -1 +1,53 @@
-# Bookstore-App
\ No newline at end of file
+## Bookstore-App
+
+## Getting Started
+Math Magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+Make simple calculations.
+Read a random math-related quote.
+
```

Comment:
- [ ] Please update the readme file description with the proper one. Try to present the "Bookstore" project rather than the "Math magicians" project.